webapp/package.json

```diff
@@ -50,8 +50,8 @@
     "pino": "^9.2.0",
     "pino-http": "^10.2.0",
     "postgres": "^3.4.4",
-    "react": "19.0.0-rc-378b305958-20240710",
-    "react-dom": "19.0.0-rc-378b305958-20240710",
+    "react": "18.2.0",
+    "react-dom": "18.2.0",
```

Comment:
I don't know why were using release candidate versions of react. I just downgraded this to avoid having tons of warnings. Everything seems to be fine after this.